python/ray/tune/logger.py

```diff
@@ -9,6 +9,7 @@
 import os
 import yaml
 
+from ray.cloudpickle import cloudpickle
```

Comment:
Can you change this to `import ray.cloudpickle as pickle` or `import ray.cloudpickle as cloudpickle`.

Reply:
yeah of course, sorry for the deviation from the standards